styles/typography.scss

```diff
@@ -23,3 +23,27 @@ code {
 .reveal-on-hover-parent:hover .reveal-on-hover-child {
   visibility: visible;
 }
+
+// Table of contents styles
```

Comment:
For the record, I don't think this is a great pattern — but it will at least address the styling concerns until we decide on a larger approach for styling. 

By not great, I just mean I don't personally think it will be great to add a bunch of bespoke styles at the top level. But happy to say more when we have that top-level discussion.

Reply:
Sounds good.

As an alternative approach to the media query I was wondering if we can take advantage of one of the [Bootstrap responsive classes to hide elments.](https://getbootstrap.com/docs/5.0/utilities/display/#hiding-elements) - but as you said, since we're going to be deciding on a styling approach in the future, it's fine as it is.